src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -6,7 +6,7 @@
 public class SalaryInfo {
 
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        DateTimeFormatter formatDate = DateTimeFormatter.ofPattern("d.MM.yyyy");
+        final DateTimeFormatter formatDate = DateTimeFormatter.ofPattern("d.MM.yyyy");
```

Comment:
it's not a constant, try to google what is constant in java and how to declare it

Reply:
got it